scripts/sync-algolia-index.ts

```diff
@@ -48,7 +51,7 @@ const getFileInfos = (
 };
 
 type AlgoliaRecord = {
-  objectId: string;
+  objectId: undefined;
   content: string;
   objectID: string;
```

Comment:
I’ve been getting a type error here because of what we do on line 68 (`objectId: undefined,`). @akash-joshi what’s the difference between `objectId` and `objectID`? I want to keep logic changes minimal, but happy to add a comment or tweak types a bit more.

Reply:
`objectId` was an old key holding auto-generated `objectID`s for each Algolia record (#37). This is now deprecated by the path-based `objectID` generation PR (#49), hence the old `objectId`s are set to undefined.